python/MooseDocs/tree/html.py

```diff
@@ -101,7 +101,7 @@ class String(NodeBase):
     """
     def __init__(self, parent=None, **kwargs):
         kwargs.setdefault('content', '')
-        kwargs.setdefault('escape', '')
+        kwargs.setdefault('escape', False)
```

Comment:
this should not be included

Reply:
I have no idea how this file was edited?

Reply:
it got changed by another commit which you must have squashed in yours

Reply:
Hmmm, anyway... fixed it.